### DISCUSSION TITLE
Please provide possibility for ignore values higher or below a value for History Graph Card

### DISCUSSION THREAD

**Discussion by gegy1** [home-assistant/frontend#14987] [Dashboard] [answered]
Hey there! Some times sensors deploying garbage data, which shows mess in the history graph data. E.g. A faulty temperature sensor shows temperature from hell (6000 C°) after a minute oder two it works again, but now the history graph shows a high rash on the graph.

**Comment by ildar170975** [comment, marked as answer]
“Bad” values are better to be filtered on a template sensor’s level.

**Reply by gegy1** [reply]
Of course that could be a solution. Where do i have to request for a feature to filter "bad" data on sensor level or is this already a feature in homeassistant?

**Comment by gegy1** [comment]
https://www.home-assistant.io/integrations/filter/#outlier
